app/services/admin/npq_applications/edge_case_search.rb

```diff
@@ -37,6 +45,30 @@ def search_conditions
     end
   end
 
+  def funding_eligiblity_status_code_conditions
```

Comment:
Was going to say all these methods are very similar and could be dried up but i think this is fine for just 3.

Reply:
I did try to do it like this in the beginning which was a much cleaner way ![image](https://github.com/DFE-Digital/early-careers-framework/assets/132997928/2cb28ba0-a39b-42d6-93d0-663853084eb8)
But didn't get it running due to the way the flow is set up. Kept on getting the following error.
![image](https://github.com/DFE-Digital/early-careers-framework/assets/132997928/e8dd75e1-07e1-4fa5-b5ee-bd6130dbca4f)
I can move them to a separate helper/concern class and call it from there but it's just three methods so i left them here.